ngraph/src/ngraph/frontend/onnx_import/op/acosh.hpp

```diff
@@ -27,7 +27,10 @@ namespace ngraph
         {
             namespace set_1
             {
```

Comment:
You may want to `#include "default_opset.hpp"` here, so we don't run into compilation issues on an exotic platform.